src/Symfony/Component/Console/Helper/SymfonyQuestionHelper.php

```diff
@@ -52,9 +53,13 @@ public function ask(InputInterface $input, OutputInterface $output, Question $qu
      */
     protected function writePrompt(OutputInterface $output, Question $question)
     {
-        $text = $question->getQuestion();
+        $text = OutputFormatter::escape($question->getQuestion());
         $default = $question->getDefault();
 
+        if (null !== $default) {
```

Comment:
Needs to be skipped for `ConfirmationQuestion`, else bool is converted to a string.